tests/e2e/streams/streams_tests.py

```diff
@@ -342,85 +367,137 @@ def test_check_already_started_stream(topics, connection):
 
     common.start_stream(cursor, "test_stream")
     time.sleep(1)
-    producer.send(topics[0], b'CREATE (n:VERTEX { id : 42 })')
-    assert common.check_one_result_row(
-        cursor, "MATCH (n:VERTEX { id : 42 }) RETURN n")
+    producer.send(topics[0], b"CREATE (n:VERTEX { id : 42 })")
+    assert common.check_one_result_row(cursor, "MATCH (n:VERTEX { id : 42 }) RETURN n")
 
 
 @pytest.mark.parametrize("transformation", TRANSFORMATIONS_TO_CHECK)
 def test_bootstrap_server(producer, topics, connection, transformation):
     assert len(topics) > 0
     cursor = connection.cursor()
     local = "localhost:9092"
-    common.execute_and_fetch_all(cursor,
-                                 "CREATE STREAM test "
-                                 f"TOPICS {','.join(topics)} "
-                                 f"TRANSFORM {transformation} "
-                                 f"BOOTSTRAP_SERVERS \'{local}\'")
+    common.execute_and_fetch_all(
+        cursor,
+        "CREATE STREAM test "
+        f"TOPICS {','.join(topics)} "
+        f"TRANSFORM {transformation} "
+        f"BOOTSTRAP_SERVERS '{local}'",
+    )
     common.start_stream(cursor, "test")
     time.sleep(5)
 
     for topic in topics:
         producer.send(topic, SIMPLE_MSG).get(timeout=60)
 
     for topic in topics:
-        common.check_vertex_exists_with_topic_and_payload(
-            cursor, topic, SIMPLE_MSG)
+        common.check_vertex_exists_with_topic_and_payload(cursor, topic, SIMPLE_MSG)
 
 
 @pytest.mark.parametrize("transformation", TRANSFORMATIONS_TO_CHECK)
 def test_bootstrap_server_empty(producer, topics, connection, transformation):
     assert len(topics) > 0
     cursor = connection.cursor()
     with pytest.raises(mgclient.DatabaseError):
-        common.execute_and_fetch_all(cursor,
-                                     "CREATE STREAM test "
-                                     f"TOPICS {','.join(topics)} "
-                                     f"TRANSFORM {transformation} "
-                                     "BOOTSTRAP_SERVERS ''")
+        common.execute_and_fetch_all(
+            cursor,
+            "CREATE STREAM test "
+            f"TOPICS {','.join(topics)} "
+            f"TRANSFORM {transformation} "
+            "BOOTSTRAP_SERVERS ''",
+        )
+
+
+@pytest.mark.parametrize("transformation", TRANSFORMATIONS_TO_CHECK)
+def test_set_offset(producer, topics, connection, transformation):
+    assert len(topics) > 0
+    cursor = connection.cursor()
+    common.execute_and_fetch_all(
+        cursor,
+        "CREATE STREAM test "
+        f"TOPICS {topics[0]} "
+        f"TRANSFORM {transformation} "
+        "BATCH_SIZE 1",
+    )
+
+    messages = [f"{i} message" for i in range(1, 21)]
+
+    for message in messages:
+        producer.send(topics[0], message.encode()).get(timeout=60)
+
+    def execute_set_offset_and_consume(id):
+        common.execute_and_fetch_all(
+            cursor, f"CALL mg.set_stream_offset('test', {id}) YIELD result"
```

Comment:
What does `YIELD result` means here? As far as I can see, `set_stream_offset` doesn't have a return value.

Reply:
that was mistyped and removed